fixtures/view-transition/README.md

```diff
@@ -28,3 +28,11 @@ yarn start:prod
 ```
 
 This will pre-build all static resources and then start a server-side rendering HTTP server that hosts the React app and service the static resources (without hot reloading).
+
+## Deploy
+
+Deployed to https://react-fixture-view-transition-six.vercel.app
```

Comment:
I don't like the `-six` suffix. Maybe I accidentally sit on `react-fixture-view-transition.vercel.app`. I'll double check.